src/development/platform-integration/platform-channels.md

```diff
@@ -542,15 +543,15 @@ tied to the channel name `samples.flutter.io/battery`:
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
 
-    let controller : FlutterViewController = window?.rootViewController as! FlutterViewController;
-    let batteryChannel = FlutterMethodChannel.init(name: "samples.flutter.io/battery",
-                                                   binaryMessenger: controller);
+    let controller : FlutterViewController = window?.rootViewController as! FlutterViewController
+    let batteryChannel = FlutterMethodChannel(name: "samples.flutter.io/battery",
```

Comment:
Isn't swift typically formatted with the colons (`:`) all aligned?

Reply:
That's not uncommon in Objective-C, but I don't see that typically done in Swift.  The [Google Swift style guidelines](https://google.github.io/swift/#horizontal-alignment) say not to do horizontal alignment.  That section doesn't specifically talk about function declarations or function calls, but there are some code examples (Trailing Closures, Delegate Methods) that show no effort to align colons.